playwright/e2e/suites/accession.spec.ts

```diff
@@ -0,0 +1,222 @@
+import { test, expect } from '@playwright/test';
+import { waitFor } from '../utils/utils';
+
+test.setTimeout(60000);
+test.beforeEach(async ({ context }, testInfo) => {
+  // Make all requests look like they are associated with an existing login session
+  // so we don't have to depend on a Keycloak server to run the test suite. The
+  // session value here is the base64-encoded session ID from dump/session.sql.
+  await context.addCookies([
```

Comment:
suggestion, move this into utils

const addCookies = async (context) => {
  await context.addCookies(....);
}

each test need only await on this imported addCookies without specifying the cookie/url details